src/main/java/de/qabel/desktop/ui/remotefs/RemoteFSController.java

```diff
@@ -32,10 +34,10 @@
 
 public class RemoteFSController extends AbstractController implements Initializable {
 
-    private static final String PRIVATE_KEY = "77076d0a7318a57d3c16c17251b26645df4c2f87ebc0992ab177fba51db92c2a";
-    public static final String ROOT_FOLDER_NAME = "root Folder";
-    private final String bucket = "qabel";
-    private final String prefix = "qabelTest";
+    private String PRIVATE_KEY;
+    public String ROOT_FOLDER_NAME = "RootFolder";
```

Comment:
If they are not static, then [LCC](https://en.wikipedia.org/wiki/CamelCase#Variations_and_synonyms) please.
Additionally, no public properties please. Package private if you need it for tests, protected if they should be part of the inheritance-API and private by default.